src/index.js

```diff
@@ -150,13 +152,14 @@ class LumigoPlugin {
 		}
 	}
 
-	async installLumigoNodejs() {
-		this.log("installing @lumigo/tracer...");
+	async installLumigoNodejs(pinVersion) {
+		const finalVersion = pinVersion ? pinVersion : "latest";
```

Comment:
let's do `const finalVersion = pinVersion || "latest"`, it's more idiomatic in Node